src/renderer/i18n/de/wallet.ts

```diff
@@ -23,7 +23,7 @@ const wallet: WalletMessages = {
   'wallet.empty.action.create': 'Erstellen',
   'wallet.empty.phrase.import': 'Importiere eine bestehende Wallet mit bestehendem Guthbaben',
   'wallet.empty.phrase.create': 'Erstelle eine neue Wallet und füge ein Guthaben hinzu',
-  'wallet.create.copy.phrase': 'Hier klicken um die Phrase zu kopieren',
+  'wallet.create.copy.phrase': 'Kopieer onderstaande zin',
```

Comment:
^ That's Dutch, but not German :smile:  

German: `Phrase kopieren`

Reply:
hmmmm.... they are different.... interesting....